src/features/unified-share-modal/__tests__/UnifiedShareForm.test.js

```diff
@@ -106,6 +106,7 @@ describe('features/unified-share-modal/UnifiedShareForm', () => {
                 shouldRenderFTUXTooltip: true,
                 showCalloutForUser: true,
             });
+            expect(wrapper.find('.usm-ftux-tooltip').prop('position')).toBe('bottom-center');
```

Comment:
not sure the best practice on this but I've generally seen us use the component name unless we need the extra specificity of the class